proto/zitadel/settings/v2/settings_service.proto

```diff
@@ -427,6 +430,97 @@ service SettingsService {
       }
     };
   }
+
+  // Set Organization Settings
+  //
+  // Sets the settings specific to an organization.
+  //
+  // Required permissions:
+  //   - `iam.policy.write`
+  rpc SetOrganizationSettings(SetOrganizationSettingsRequest) returns (SetOrganizationSettingsResponse) {
```

Comment:
IMO stating just `settings` in the description feels a bit vague. I'd mention what settings even if it's just one at the moment.

Reply:
I left it vague with the intention of that there is a possibility to add other settings to it.
Can add it as a list for example.